GliaWidgets/View/Call/CallStyle.swift

```diff
@@ -83,7 +91,9 @@ public class CallStyle: EngagementStyle {
         bottomText: String,
         bottomTextFont: UIFont,
         bottomTextColor: UIColor,
-        buttonBar: CallButtonBarStyle
+        buttonBar: CallButtonBarStyle,
+        onHoldTopText: String,
+        onHoldBottomText: String
```

Comment:
Since we have a pretty big constructor already, perhaps it makes sense to create a separate structure `OnHold` and place there two properties for now?